modules/csvcreator/CMakeLists.txt

```diff
@@ -1,2 +1,6 @@
-include_directories(include)
+include_directories(include ../core/include)
+ADD_DEFINITIONS("-DBOOST_LOG_DYN_LINK")
 add_library(CSVCreator src/csv.cpp)
+target_link_libraries(CSVCreator ${OpenCV_LIBS} ${Boost_LIBRARIES} Core)
+target_compile_features(CSVCreator PRIVATE cxx_range_for)
```

Comment:
For whatever reason, the targetanalysis module needs it too.
When it compiles without the line, it errors during a file read.